### ISSUE TITLE
ST-26072018 - Definition component

### ISSUE THREAD

**Issue by armstrongb** [ONSdigital/sdc-global-design-patterns#143]
### Expected behaviour
The link should announce as being ‘collapsed’ when closed using the ‘hide’ button.

From the audit report:
>The ‘hide this’ button announces that it is collapsed. A button should not have a collapsed or expanded state.
>If the definition is closed using the title, then the link goes back to being announced as ‘...collapsed...’. If the ‘hide this’ button is used, the link is announced as ‘...expanded...’ even though it is visually collapsed.
### Actual behaviour
>If the ‘hide this’ button is used, the link is announced as ‘...expanded...’ even though it is visually collapsed

### Steps to reproduce the behaviour
Use VoiceOver or other browser to test the definition component.

**Comment by armstrongb** [comment]
Closing as resolved in [v1.5.2](https://github.com/ONSdigital/sdc-global-design-patterns/releases/tag/v1.5.2)
